Explorer/Assets/DCL/PluginSystem/World/TweenPlugin.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Arch.SystemGroups;
+using Cysharp.Threading.Tasks;
+using DCL.ECSComponents;
+using DCL.PluginSystem.World.Dependencies;
+using DCL.Utilities;
+using ECS.LifeCycle;
+using ECS.LifeCycle.Systems;
+using ECS.Unity.Tween.Systems;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace DCL.PluginSystem.World
+{
+    public class TweenPlugin : IDCLWorldPlugin
+    {
+        private readonly WorldProxy globalWorld;
+
+        public TweenPlugin(WorldProxy globalWorld)
+        {
+            this.globalWorld = globalWorld;
+        }
```

Comment:
I think you don't need the global world at all for the Tween component, you could remove it and all of its references.

In other components that was needed to propagate some stuff from the global world into the scene-level world, because we have some plugins at global level instead of at scene-level (like the AvatarPlugin, needed for the AvatarShapePlugin that runs at scene-world-level)), since those 2 different worlds (global/scene) are separated, they manage a different set of entities.

Your `globalWorldEntity` can just be a regular Entity from the system's `World` (the scene-level world).

Reply:
you are right! thank you